src/main/resources/views/topicProduce.ftl

```diff
@@ -19,6 +19,12 @@
             </select>
         </div>
     </div>
+    <div class="form-group row">
+        <label for="key-schema" class="col-sm-2 col-form-label">Key schema id</label>
+        <div class="col-sm-10">
+            <input type="text" class="form-control" name="keySchema" id="key-schema" autocomplete="off" placeholder="Key Schema id">
```

Comment:
Please change this to a combo box like here : 
https://github.com/tchiotludo/kafkahq/blob/fdac9438b7c64bab1e1905dbe7dcc69e23caf4f3/src/main/resources/views/tail.ftl#L36-L45
with the list of available schema on current registry instead of id

Reply:
No problem. 
Do you want all the schemas or just the last version of each subject?

Reply:
I would say the last version of the schema, dont think people use outdated schema most of the time. 
Also if it's possible to sort the list and display the corresponding schema on top from this topic (example topic name: my-topic / display on top my-topic-value or key schema) ?

Reply:
Alright. Great idea for the sorting :+1: (although only useful when using topic-named strategy)

Reply:
I think most of people use topic-named strategy and will be happy.
Others just have to search in the combo :laughing: